packages/styling/src/styles/theme.ts

```diff
@@ -53,6 +80,14 @@ export function loadTheme(theme: IPartialTheme): ITheme {
 
   Customizations.applySettings({ [ThemeSettingName]: _theme });
 
+  for (let callback of _onThemeChangeCallbacks) {
```

Comment:
nit: const callback

Reply:
it's a let since the var name stays the same but the value changes, I don't think it's recreating it each loop

Reply:
Foreach instead of for of?

Reply:
what's the difference?

Reply:
I'd just prefer we be consistent in the code base, they are pretty similar. foreach can optionally also provide the index, so it tends to be more flexible and it is easier to read IMO.

I don't think we have a solid consensus here in fabric, so consider this a nit comment that you can safely ignore